kymatio/scattering2d/tests/test_scattering2d.py

```diff
@@ -261,3 +261,32 @@ def test_batch_shape_agnostic():
         assert Sx.shape[-2:] == shape_ds
         assert Sx.shape[-3] == n_coeffs
         assert Sx.shape[:-3] == test_shape[:-2]
+
```

Comment:
Maybe add a one line comment to describe the function and then it is good for merging imho

Reply:
Ok will do.